raster_api/infrastructure/construct.py

```diff
@@ -89,7 +90,7 @@ def __init__(
             ] = aws_apigatewayv2_alpha.ParameterMapping().overwrite_header(
                 "host",
                 aws_apigatewayv2_alpha.MappingValue(
-                    veda_raster_settings.domain_hosted_zone_name
+                    f"{stage}.{veda_raster_settings.domain_hosted_zone_name}"
```

Comment:
comment for future work: maybe we can add a method in the api config files to add a stage subdomain for all non-prod stages? We don't need that now so it's just something to think about.